ui/components/multichain/account-overview/account-overview-layout.tsx

```diff
@@ -94,6 +97,10 @@ export const AccountOverviewLayout = ({
     }
     ///: END:ONLY_INCLUDE_IF
 
+    if (id === 'smartAccountUpgrade') {
```

Comment:
Minor, do we need logic for these click handlers?

Reply:
Ideally yep, but it is bit hard as there are no tests for carousel element currently.